ios/RCTWebRTC/WebRTCModule+DailyDevicesManager.h

```diff
@@ -7,15 +7,20 @@
 
 #import "WebRTCModule.h"
 
+// audio devices routes
 static NSString * _Nonnull const WIRED_OR_EARPIECE_DEVICE_ID = @"WIRED_OR_EARPIECE";
 static NSString * _Nonnull const SPEAKERPHONE_DEVICE_ID = @"SPEAKERPHONE";
 static NSString * _Nonnull const BLUETOOTH_DEVICE_ID = @"BLUETOOTH";
 
 @interface WebRTCModule (DailyDevicesManager)
 
+@property (nonatomic, strong) NSString *userSelectedDevice;
+
 - (BOOL)hasBluetoothDevice;
 - (BOOL)hasWiredHeadset;
-
 - (void)setAudioDevice:(nonnull NSString*)deviceId;
+- (void)configureAudioSession:(AVAudioSessionCategoryOptions) categoryOptions
```

Comment:
The method name should reflect the arguments to be proper Objective-C. In this case, `configureAudioSession:` implies the first argument should be an audio session, which it's not.

I also noticed that this method is "public" (it's in the header file) and is next to `setAudioDevice:`, which *doesn't* need an `AVAudioSession` passed in.

For parallelism, I'd recommend making a "public" method here with the signature: `setAudioMode:options:`. Internally, that method can call `configureAudioSession:withAudioMode:options:` (which has its arguments rearranged so that it reads well).

Reply:
Refactor to the audioSession to be the first argument of the method. This method is only used between the categories.